### PR TITLE
Reactivating testnet3 related tests

### PR DESCRIPTION
**Pull request by jleni**

### CODE REVIEW

**Comment by jleni** [comment]
This is expected. CI is currently using a testnet2 endpoint. I will update CI to use a testnet3 endpoint from now on.

File: signer/src/api.rs [hunk 1]
```diff
@@ -82,11 +87,11 @@ impl From<&Signature> for SignatureAPI {
     fn from(sig: &Signature) -> SignatureAPI {
         match sig {
             Signature::SignatureSECP256K1(sig_secp256k1) => SignatureAPI {
-                sig_type: "secp256k1".to_string(),
+                sig_type: SigTypes::SigTypeSecp256k1 as u8,
```
Comment (jleni) [inline signer/src/api.rs:90]: Oh I see.. this is actually a breaking change..!

File: signer/src/lib.rs [hunk 2]
```diff
@@ -320,7 +320,7 @@ fn verify_secp256k1_signature(
     let blob_to_sign = Message::parse_slice(&message_digest)?;
 
     let public_key = recover(&blob_to_sign, &signature_rs, &recovery_id)?;
-    let mut from = Address::new_secp256k1(&public_key.serialize_compressed().to_vec());
+    let mut from = Address::new_secp256k1(&public_key.serialize().to_vec());
```
Comment (jleni) [inline signer/src/lib.rs:323]: this is also a breaking change... mmm... are we sure about this? There were some problems in the past because of this...

Comment (jleni) [reply signer/src/lib.rs:323]: yes, now I remember. This is definitely a bug.
I had trouble in the past, and requested the spec team to make this explicit

https://github.com/filecoin-project/specs/pull/872#issue-369581772

Comment (rllola) [reply signer/src/lib.rs:323]: Yeah that was weird. That's why also I had to change tests were I was verifying signatures. They should check length.

It was actually the only place where it was using compress key so no big deal.

Comment (jleni) [reply signer/src/lib.rs:323]: Austin did it already :)
https://github.com/ChainSafe/forest/pull/438

**Comment by jleni** [comment]
Great work! :100:

**Pull request merged**